source/docs/contracts_configuration.md

```diff
@@ -281,9 +281,54 @@ contracts: {
 </code></pre>
 
 ## Deployment tracking
-Embark's Smart Contract deployment mechanism prevents the deployment of Smart Contracts that have already been deployed. This turns out to be a powerful feature as you don't have to worry about keeping track of it. If we prefer to have full control over the deployment process and don't want Embark to keep track of individual Smart Contracts deployments, we use the `track` configuration and set it `false`.
 
-The following example ensure `ERC20` won't be tracked and therefore redeployed in every deployment cycle.
+Embark's Smart Contract deployment mechanism prevents the deployment of Smart Contracts that have already been deployed. This turns out to be a powerful feature as you don't have to worry about keeping track of it. The way this works is that, by default, Embark creates a file `./embark/chains.json` in which it stores the name and address of the deployed Smart Contracts. That information is than mapped to the hash of the block in which the Smart Contract have been deployed:
```

Comment:
```suggestion
Embark's Smart Contract deployment mechanism prevents the deployment of Smart Contracts that have already been deployed. This turns out to be a powerful feature as you don't have to worry about keeping track of it. The way this works is that, by default, Embark creates a file `./embark/chains.json` in which it stores the name and address of the deployed Smart Contracts. That information is then mapped to the hash of the block in which the Smart Contract have been deployed:
```

Reply:
I'm curious about the path `./embark/chains.json`. Isn't `chains.json` created at the top-level of the project directory?

Reply:
Nope, it lives inside `.embark/`